resources/metadata/helmExecute.yaml

```diff
@@ -0,0 +1,371 @@
+
+          Recommendation is to use custom values and pass them via parameter `helmValues`.<br />
+          Since helm supports multiple files on top of the `values.yaml`, landscape-specific attributes can be passed via a specific file.
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: keepFailedDeployments
+        type: bool
+        description: Defines whether a failed deployment will be purged
+        default: false
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: kubeConfig
+        type: string
+        description: Defines the path to the "kubeconfig" file.
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        secret: true
+        resourceRef:
+          - name: kubeConfigFileCredentialsId
+            type: secret
+          - type: vaultSecretFile
+            name: kubeConfigFileVaultSecretName
+            default: kube-config
+      - name: kubeContext
+        type: string
+        description: Defines the context to use from the "kubeconfig" file.
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      # - name: kubeToken
+      #   type: string
+      #   description: Contains the id_token used by kubectl for authentication. Consider using kubeConfig parameter instead.
+      #   scope:
+      #     - GENERAL
+      #     - PARAMETERS
+      #     - STAGES
+      #     - STEPS
+      #   secret: true
+      #   resourceRef:
+      #     - name: kubeTokenCredentialsId
+      #       type: secret
+      - name: namespace
+        aliases:
+          - name: helmDeploymentNamespace
+          - name: k8sDeploymentNamespace
+        type: string
+        description: Defines the target Kubernetes namespace for the deployment.
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        default: default
+      - name: tillerNamespace
+        aliases:
+          - name: helmTillerNamespace
+        type: string
+        description: Defines optional tiller namespace for deployments using helm.
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: dockerConfigJSON
+        type: string
+        description: Path to the file `.docker/config.json` - this is typically provided by your CI/CD system. You can find more details about the Docker credentials in the [Docker documentation](https://docs.docker.com/engine/reference/commandline/login/).
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        secret: true
+        resourceRef:
+          - name: dockerConfigJsonCredentialsId
+            type: secret
+          - type: vaultSecretFile
+            name: dockerConfigFileVaultSecretName
+            default: docker-config
+      - name: deployCommand
```

Comment:
step goes beyond deployment, thus I guess we need a rather general parameter name